lms/djangoapps/course_api/blocks/transformers/tests/test_milestones.py

```diff
@@ -19,7 +20,7 @@
 @attr('shard_3')
 @ddt.ddt
 @patch.dict('django.conf.settings.FEATURES', {'ENABLE_SPECIAL_EXAMS': True, 'MILESTONES_APP': True})
-class MilestonesTransformerTestCase(CourseStructureTestCase, MilestonesTestCaseMixin):
+class MilestonesTransformerTestCase(CourseStructureTestCase, MilestonesTestCaseMixin, ModuleStoreTestCase):
```

Comment:
Please remove `ModuleStoreTestCase` since `CourseStructureTestCase` already inherits from `ModuleStoreTestCase`. 
Also, keep in mind that test classes can inherit from many Mixins, but only one TestCase class.